PermanentOrg/web-app
# PER - 9235

I have added another api call for the profile items and a spinner, this way the modal opens each time

Steps to test:If you are logged in:
 Go to any public archive then refresh the page,  then go to your private archive by clicking your round image in the top bar, then click the archive profile. It should open

If you are not logged in: When viewing any public archive, go to sign up, then after being authenticated go to archive profile.
